components/state/src/index.js

```diff
@@ -68,13 +68,14 @@ AFRAME.registerSystem('state', {
    * Dispatch action.
    */
   dispatch: (function () {
-    const toUpdate = [];
 
     return function (actionName, payload) {
       var dirtyArrays;
       var i;
       var key;
       var subscription;
+      
+      const toUpdate = [];
```

Comment:
possible to keep this where it was? it was to prevent array allocation on each action

Reply:
good point, fixed